public/pattern-library/data.json

```diff
@@ -0,0 +1,128 @@
+{
+    "project_logo": null,
+    "project_favicon": "ryr_logo.png",
+    "project_name": "Request Yo Racks",
+    "project_url": null,
```

Comment:
You can put the github url until we decide to publish something: http://request-yo-racks.github.io/.